lib/Transforms/Scalar/DxilLoopUnroll.cpp

```diff
@@ -724,6 +725,27 @@ void DxilLoopUnroll::RecursivelyRecreateSubLoopForIteration(LPPassManager &LPM,
   }
 }
 
+static void RemapDebugInsts(BasicBlock *ClonedBB, ValueToValueMapTy &VarMap, SetVector<BasicBlock *> &ClonedFrom) {
+  LLVMContext &Ctx = ClonedBB->getContext();
+  for (Instruction &I : *ClonedBB) {
+    DbgValueInst *DV = dyn_cast<DbgValueInst>(&I);
+    if (!DV)
+      continue;
+
+    Instruction *I = dyn_cast_or_null<Instruction>(DV->getValue());
```

Comment:
Oh yeah, this results in error on linux builds due to redefinition of `I` with different types.